src/index.js

```diff
@@ -40,36 +41,49 @@ export { default as Utils } from './utils';
  * - MDN: [HTMLTemplateElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLTemplateElement)
  */
 
-/*
- * Register element definitions with the Custom Element registry.
- */
-function _defineElements () {
-    for (let element in Elements) {
-        Elements[element].$define();
-    }
-}
+const HelixUI = {
+    Elements,
+    Utils,
+    VERSION,
 
-/**
- * Initialize HelixUI when Web Components are ready.
- */
-export function initialize () {
-    if (window.WebComponents) {
-        // Polyfill detected
-        if (window.WebComponents.ready) {
-            // polyfill already finished loading, initialize immediately
-            _defineElements();
+    /**
+     * Initialize HelixUI when Web Components are ready.
+     * (synchronous)
+     *
+     * @param {Function?} cb - callback function to execute once HelixUI is fully initialized.
```

Comment:
Look up jsdoc syntax for optional params. I'm not sure `{Function?}` is proper syntax.